### PR TITLE
Mention Prometheus Operator in Kubernetes SD section.

### PR DESCRIPTION
**Pull request by brancz**
We repeatedly get requests to solve problems for/with people that can easily be solved with the referenced resources.

I understand that this might be advertising a third party a bit but it also solves a problem the community has.

Feedback and opinions welcome.

@fabxc @brian-brazil @juliusv @beorn7

### CODE REVIEW

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -508,6 +508,13 @@ Kubernetes SD configurations allow retrieving scrape targets from
 [Kubernetes'](http://kubernetes.io/) REST API and always staying synchronized with
 the cluster state.
 
+> Make sure to check out the [Prometheus
```
Comment (brancz) [inline content/docs/operating/configuration.md]: Moved it to the sample and changed the wording a bit.

File: content/docs/operating/configuration.md [hunk 2]
```diff
@@ -621,7 +621,12 @@ tls_config:
 Where `<role>` must be `endpoints`, `service`, `pod`, or `node`.
 
 See [this example Prometheus configuration file](https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml)
-for a detailed example of configuring Prometheus for Kubernetes.
+for a detailed example of configuring Prometheus for Kubernetes. Also check out
+the [Prometheus Operator](https://github.com/coreos/prometheus-operator) and
+[kube-prometheus](https://github.com/coreos/kube-prometheus) when using
+Prometheus with Kubernetes. It automates your Prometheus setup on top of
+Kubernetes and does not require you to learn the configuration idioms of
+Prometheus.
```
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: I'd go for something like:

You may wish to check out the 3rd party Prometheus Operator, which automates the Prometheus setup on top of Kubernetes.

Comment (beorn7) [inline content/docs/operating/configuration.md]: Even more concise. However, https://github.com/coreos/kube-prometheus links to https://github.com/coreos/prometheus-operator , but not vice versa. So if we want to give only one pointer, it should be to https://github.com/coreos/kube-prometheus.

Comment (brancz) [inline content/docs/operating/configuration.md]: I'm [adding](https://github.com/coreos/prometheus-operator/pull/60) a mention of kube-prometheus to the Prometheus Operator. Also updated the wording accordingly.

**Comment by beorn7** [comment]
:+1:

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**